proposals/README.md

```diff
@@ -0,0 +1,241 @@
+        }
+
+        assertEq(maxSupplyTotal, 100_420, "Invalid maxSupplyTotal");
+    }
+
+    function _beforeDeploy(Addresses, address deployer) internal override {
+        setAndConfirmWearableData();
+    }
+
+    function _deploy(Addresses addresses, address) internal override {}
+
+    function _afterDeploy(Addresses addresses, address) internal override {}
+
+    function _afterDeployOnChain(Addresses, address deployer) internal virtual override {}
+
+    function _aferDeployForTestingOnly(Addresses, address deployer) internal virtual override {}
+
+    function _teardown(Addresses addresses, address deployer) internal override {}
+
+    function _build(Addresses addresses, address) internal override {
+        /// Wearable config
+        address wearables = addresses.getAddress("ERC1155_MAX_SUPPLY_MINTABLE_WEARABLES");
+        for (uint256 i = 0; i < wearableTokenIDMaxSupplySettings.length; i++) {
+            _pushTimelockAction(
+                wearables,
+                abi.encodeWithSignature(
+                    "setSupplyCap(uint256,uint256)",
+                    wearableTokenIDMaxSupplySettings[i].tokenId,
+                    wearableTokenIDMaxSupplySettings[i].maxSupply
+                ),
+                string(
+                    abi.encodePacked(
+                        "Set wearable tokenId ",
+                        wearableTokenIDMaxSupplySettings[i].tokenId,
+                        " to max supply ",
+                        wearableTokenIDMaxSupplySettings[i].maxSupply
+                    )
+                )
+            );
+        }
+    }
+
+    function _run(Addresses addresses, address) internal override {
+        this.setDebug(true);
+
+        _simulateTimelockActions(
+            addresses.getAddress("ADMIN_TIMELOCK_CONTROLLER"),
+            addresses.getAddress("ADMIN_MULTISIG"),
+            addresses.getAddress("ADMIN_MULTISIG")
+        );
+    }
+
+    function _validate(Addresses addresses, address) internal override {
+        ERC1155MaxSupplyMintable wearable = ERC1155MaxSupplyMintable(
+            addresses.getAddress("ERC1155_MAX_SUPPLY_MINTABLE_WEARABLES")
+        );
+
+        /// Verfiy Wearable
+        for (uint256 i = 0; i < wearableTokenIDMaxSupplySettings.length; i++) {
+            uint256 tokenId = wearableTokenIDMaxSupplySettings[i].tokenId;
+            uint256 maxSupply = wearableTokenIDMaxSupplySettings[i].maxSupply;
+            uint256 currentSupply = wearable.totalSupply(tokenId);
+
+            assertEq(wearable.maxTokenSupply(tokenId), maxSupply, "Invalid maxTokenSupply for tokenId");
+            assertEq(wearable.getMintAmountLeft(tokenId), maxSupply - currentSupply, "Invalid getMintAmountLeft for tokenId");
+        }
+    }
+
+    function _validateOnChain(Addresses, address deployer) internal virtual override {}
+
+    function _validateForTestingOnly(Addresses, address deployer) internal virtual override {}
+}
+```
```

Comment:
Do we really need to include this? Could we link to an example file instead?

Reply:
We don't need this. removed it.